addons/website_forum/controllers/website_forum.py

```diff
@@ -179,31 +180,64 @@ def tag_read(self, forum_id, query='', limit=25, **post):
             headers=[("Content-Type", "application/json")]
         )
 
-    @http.route(['/forum/<model("forum.forum"):forum>/tag', '/forum/<model("forum.forum"):forum>/tag/<string:tag_char>'], type='http', auth="public", website=True, sitemap=False)
-    def tags(self, forum, tag_char=None, **post):
-        # build the list of tag first char, with their value as tag_char param Ex : [('All', 'all'), ('C', 'c'), ('G', 'g'), ('Z', z)]
-        first_char_tag = forum.get_tags_first_char()
-        first_char_list = [(t, t.lower()) for t in first_char_tag if t.isalnum()]
-        first_char_list.insert(0, (_('All'), 'all'))
-
-        active_char_tag = tag_char and tag_char.lower() or 'all'
-
-        # generate domain for searched tags
+    @http.route(['/forum/<model("forum.forum"):forum>/tag',
+                 '/forum/<model("forum.forum"):forum>/tag/<string:tag_char>',
+                 ], type='http', auth="public", website=True, sitemap=False)
+    def tags(self, forum, tag_char=None, filters='all', search='', **post):
+        """Render a list of tags matching filters and search parameters.
+
+        :param forum: Forum
+        :param string|none tag_char: Only tags starting with a single character `tag_char`
```

Comment:
Would support a real "starts_with", because it does not cost much to us ... the single character is just a "strange" use case (still wondering why we support that instead of simply the search).